CHANGES

```diff
@@ -381,6 +381,10 @@
      applications with zero-copy system calls such as sendfile and splice.
      [Boris Pismenny]
 
+  *) Print all values for a PKCS#12 attribute with 'openssl pkcs12', not just
+     the first value.
+     [Jon Spillett]
+
```

Comment:
To whoever merges this: move this to the top of the applicable CHANGES list.

Reply:
That's for me a matter of obviousness, like a duck takes to water ;-)

I'll merge...

Reply:
True, you have that attention for detail.

Reply:
Why is that necessary?  There are other PR's that are going to push new CHANGES entries on top of this one.  As long as they're all in the same release, does it matter?